src/__experimental__/components/input-icon-toggle/input-icon-toggle.component.js

```diff
@@ -22,14 +22,15 @@ const InputIconToggle = ({
   info,
   useValidationIcon,
   align,
+  iconTabIndex,
 }) => {
   if (
     useValidationIcon &&
     !disabled &&
     shouldDisplayValidationIcon({ error, warning, info })
   ) {
     return (
-      <InputIconToggleStyle size={size}>
+      <InputIconToggleStyle size={size} tabIndex={iconTabIndex}>
```

Comment:
I suggest to use prop `tabbable` which will be a `boolean`. It will be more friendly for the user in my opinion so the user won't think what number to use but he will could decide how it could works. 
`tabbable` prop is also used in our other components so it will be more consistent.

Reply:
I think this is OK, input-toggle-icon isnt referenced in the docs anywhere, it seems to just be used internally. We should be able to pass tabIndex as its a normal HTML attribute.